01_install_requirements.sh

```diff
@@ -73,3 +68,55 @@ if ! which dep 2>&1 >/dev/null ; then
     mkdir -p $GOPATH/bin
     curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 fi
+
+# httpd-tools provides htpasswd utility
+sudo yum install -y httpd-tools
+
+sudo mkdir -pv ${REGISTRY_DIR}/{auth,certs,data}
+sudo chown -R $USER:$USER ${REGISTRY_DIR}
+
+pushd $REGISTRY_DIR/certs
+SSL_HOST_NAME="${LOCAL_REGISTRY_ADDRESS}"
+
+if [[ $( echo $SSL_HOST_NAME | grep -Eo '^[[:digit:]]{1,3}\.[[:digit:]]{1,3}\.[[:digit:]]{1,3}\.[[:digit:]]{1,3}$') ]];then
+    SSL_EXT="subjectAltName = IP:${SSL_HOST_NAME}"
+else
+    SSL_EXT="subjectAltName = otherName:${SSL_HOST_NAME}"
+fi
+
+openssl genrsa -out ${REGISTRY_DIR}/certs/registry.key 2048
+
+openssl req -x509 \
+    -key ${REGISTRY_DIR}/certs/registry.key \
+    -out  ${REGISTRY_DIR}/certs/registry.crt \
+    -days 365 \
+    -addext "$SSL_EXT" \
```

Comment:
The `-addext` flag isn't supported by the version of openssl that is installed via dev-scripts. It was introduced in 1.1.1.

Reply:
@honza which one do U have? I'm trying on RHEL-8 which has *openssl-1.1.1-8*

Reply:
OpenSSL 1.0.2k-fips on centos 7.7.1908

Reply:
and for me on RHEL 8.1...
```
# rpm -q openssl
openssl-1.1.1c-2.el8.x86_64

# openssl version
OpenSSL 1.1.1c FIPS  28 May 2019
```

If dev-scripts is shipping openssl then that needs to stop.  Dev-scripts should necessary args nothing more.